test/test_data/gnupg.py

```diff
@@ -0,0 +1,21 @@
+mapping_test_data = [
+    {
+        "product": "gnupg",
+        "version": "2.2.27",
+        "version_strings": ["GNU Privacy Guard's OpenPGP server 2.2.27 ready"],
+    },
+    {
+        "product": "gnupg",
+        "version": "2.2.23",
+        "version_strings": ["GNU Privacy Guard's OpenPGP server 2.2.23 ready"],
+    },
```

Comment:
Did you also want to add a test for the G13 string?

Reply:
Well, I didn't have it in mind. But I'll go ahead and add it anyways.